classes/blocks/class-covers.php

```diff
@@ -44,7 +44,6 @@ public function add_block_shortcode( $attributes, $content ) {
 		);
 
 		$attributes['tags']       = explode( ',', $attributes['tags'] );
```

Comment:
Could you also adjust space here?
phpcs: `Equals sign not aligned correctly; expected 1 space but found 7 spaces`